core/aws_ddk_core/resources/_kms.py

```diff
@@ -33,6 +33,10 @@ class KeyConfigSchema(BaseSchema):
 
 
 class KMSFactory:
+    """
+    Class factory to create and configure Key Management Service DDK resources, including Keys.
```

Comment:
Class that contains factory methods to create KMS resources? Let's use the common terminology